contracts/LiquidityBridgeContract.sol

```diff
@@ -57,28 +71,42 @@ contract LiquidityBridgeContract {
     mapping(address => uint256) private balances;
     mapping(address => uint256) private collateral;
     mapping(bytes32 => Registry) private callRegistry;  
-    mapping(address => uint256) private resignations;
+    mapping(address => uint256) private resignationBlockNum;
+
+    uint64 private minCollateral;        
+    uint32 private rewardP;     
+    uint32 private resignDelayInBlocks;  
+    int256 private dust;
```

Comment:
Can you add a public getter to inspect the dust value ?

Reply:
done!